python/ray/tests/test_component_failures_3.py

```diff
@@ -58,6 +63,21 @@ def ping(self):
                     ray.get(out)
                 except ray.exceptions.RayActorError:
                     children[i] = Child.remote(death_probability)
+
+            if (RAY_FORCE_DIRECT):
+                children_out = [
+                    child.get_probability.remote() for child in children
+                ]
+                # Wait for new created actors to finish creation before
+                # removing a node. This is needed because right now we don't
+                # support reconstructing actors that died in the process of
+                # being created.
+                ready, _ = ray.wait(
+                    children_out,
+                    num_returns=len(children_out),
+                    timeout=5 * 60.0)
+                assert len(ready) == len(children_out)
```

Comment:
This is a regression right? I thought the resubmission would happen still from the owner?

Reply:
The comment is not quite accurate, what it wants to say is that raylet currently only takes care of actor reconstructions for successfully created actors. The resubmission of actor creation task from owner is still supported at the moment. Previously the test is kind of flaky, so I revised the case to focus more on created actors.

Reply:
btw: this test `test_actor_creation_node_failure` exists in both `test_component_failures_2.py` and `test_component_failures_3.py`. It is disabled for direct call in the former file, while enabled in the latter.

Reply:
Got it. So actually, we did not need to change this file since RAY_FORCE_DIRECT is always True, so this is a no-op.